locales/zh_CN/LC_MESSAGES/contributor_guide.po

```diff
@@ -155,825 +164,801 @@ msgstr "单元/集成测试是我们的好伙伴。"
+"single commit**."
+msgstr "PR 中的所有 commit 都应被 **压缩&合并到 master 分支的一个 commit 里** 。"
 
 #: ../../contributor_guide.rst:62
 msgid "PR authors **should not squash commits on their own**;"
-msgstr ""
+msgstr "为保留清晰的提交日志 PR 作者 **不应该将多条 commit 压缩(squash)后提交**;"
 
 #: ../../contributor_guide.rst:63
 msgid ""
-"When implementing a complex feature, consider breaking it down into small"
-" PRs, to keep a more detailed development history and to interact with "
-"core developers more frequently."
-msgstr "当实现一个复杂的特性时，考虑将其分为许多个小 PR，从而保证更具细节的开发时间线，保证与维护者更频繁的沟通。"
+"When implementing a complex feature, consider breaking it down into small PRs, "
+"to keep a more detailed development history and to interact with core "
+"developers more frequently."
+msgstr ""
+"当实现一个复杂的特性时，考虑将其分散为许多个小 PR，从而保证更具细节的开发时间"
+"线，保证与开发者更频繁的沟通。"
 
 #: ../../contributor_guide.rst:64
 msgid "If you want early feedback from core developers"
-msgstr ""
+msgstr "如果你想更及时的得到核心开发成员的反馈"
 
 #: ../../contributor_guide.rst:66
 msgid ""
-"Open a PR in `Draft <https://github.blog/2019-02-14-introducing-draft-"
-"pull-requests/>`_ state on GitHub so that you can share your progress;"
+"Open a PR in `Draft <https://github.blog/2019-02-14-introducing-draft-pull-"
+"requests/>`_ state on GitHub so that you can share your progress;"
 msgstr ""
+"通过 GitHub 的 `Draft <https://github.blog/2019-02-14-introducing-draft-pull-"
+"requests/>`_ 状态开一个 PR，这样就可以和我们实时分享你的进展了;"
 
 #: ../../contributor_guide.rst:67
 msgid ""
-"Make sure you @ the corresponding developer in the comments or request "
-"the review."
-msgstr ""
+"Make sure you @ the corresponding developer in the comments or request the "
+"review."
+msgstr "请确保在评论中 @ 相应开发成员，或者使用请求评审(request the review)。"
 
 #: ../../contributor_guide.rst:69
 msgid "If you are making multiple PRs"
-msgstr "如果你在做很多个 PR"
+msgstr "如果你同时在处理多个 PR"
 
 #: ../../contributor_guide.rst:71
 msgid ""
 "Independent PRs should be based on **different** branches forking from "
 "``master``;"
-msgstr "互不依赖的 PR 应该被变基到来自 ``master`` 的 **不同** 分支;"
+msgstr "互不依赖的 PR 都应该是基于 ``master`` 衍生出的 **不同** 分支;"
 
 #: ../../contributor_guide.rst:72
 msgid ""
-"PRs with dependencies should be raised only after all prerequisite PRs "
-"are merged into ``master``."
-msgstr "互相依赖的 PR 应该在所有前置 PR 合并入 ``master`` 后才可以提出。"
+"PRs with dependencies should be raised only after all prerequisite PRs are "
+"merged into ``master``."
+msgstr "互相依赖的 PR 应该在所有前置 PR 合并入 ``master`` 后再进行提出。"
 
 #: ../../contributor_guide.rst:74
 msgid "All PRs should ideally come with corresponding **tests**;"
-msgstr "所有 PR 理想情况下应该伴随着相应的 **测试**;"
+msgstr "所有 PR 理想情况下都应该伴随着相应的 **测试**;"
 
 #: ../../contributor_guide.rst:75
 msgid ""
-"All PRs should come with **documentation update**, except for internal "
-"compiler implementations;"
+"All PRs should come with **documentation update**, except for internal compiler "
+"implementations;"
 msgstr ""
+"除了内部编译器的实现外，其余的 PR 都应该带有标签 **文档更新(documentation "
+"update)**;"
```

Comment:
```suggestion
"除了内部编译器的实现外，其余的 PR 都应该带有与其功能相应的 **文档更新**;"
```
It's not a label, also not a term. e.g.: I added a function `ti.step` in a PR, then I should add the usage of `ti.step` in documentation in that PR too.